### DISCUSSION TITLE
Add option to share dashboard or graph as static image

### DISCUSSION THREAD

**Discussion by SethFalco** [grafana/grafana#53424] [Dashboards, Panels & Visualizations]
Graphs in a Grafana dashboard can be shared/embedded on other websites. However, it requires the use of an `iframe`, so it can't be used on things like a GitHub README.

I think it'd be really nice if under the **Share** tab there was also a way to generate an SVG or PNG image which could be shared instead. But not a static export like taking a screenshot, I mean a link that returns the image and updates automatically on Grafana's side.

Similar to how shields/badges work that we frequently see on GitHub READMEs. Grafana could generate a copy of the image and keep it cached when a user visits the URL, and the cache can be invalidated at whatever interval it's configured for.

### Motivation

Recently, I added Stargazers to the GitHub Datasource, so we could view the growth of the repository and how it correlates with other events.

* https://github.com/grafana/github-datasource/pull/177

I'd like to embed this information on the GitHub README/Wiki of projects I maintain, for example here:
https://github.com/tldr-pages/tldr/wiki/Project-history#popularity-boosts

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
